prowler/providers/gcp/gcp_provider.py

```diff
@@ -528,3 +550,28 @@ def validate_static_arguments(
             "refresh_token": refresh_token,
             "type": "authorized_user",
         }
+
+    @staticmethod
+    def validate_project_id(provider_id: str, credentials: str = None) -> bool:
+        """
+        Validate the provider ID
+
+        Args:
+            provider_id: str
```

Comment:
Please improve the docstring with:
- Better explanation of what the function does
- Args and their description

Reply:
Nothing to do if you remove it.

Reply:
DOne!